README.md

```diff
@@ -10,7 +10,14 @@ If you want to install the plugin on your WordPress site, you can download and i
 
 ### GitHub Releases
 
-You can also download the latest version of the plugin from the [GitHub releases page](https://github.com/actblue/actblue_wordpress_plugin/releases). These releases are signed with the [ActBlue GPG key](#ActBlue-GPG-key).
+You can also download the latest version of the plugin from the [GitHub releases page](https://github.com/actblue/actblue_wordpress_plugin/releases).
+
+#### Verifying GitHub Releases
+
+These releases are signed with the [ActBlue GPG key](#ActBlue-GPG-key), to verify the signature:
+
+1. Import the signing key: `gpg --keyserver pgp.mit.edu --recv-keys D842AF7166B99EDA`
+2. Verify the downloaded file: `gpg --verify actblue-contributions.zip.asc`
```

Comment:
You should also indicate what the output should be/what to look for if the signature is good.